src/drivers/uploadthing.ts

```diff
@@ -4,87 +4,111 @@ import { UTApi } from "uploadthing/server";
 
 export interface UploadThingOptions {
   apiKey: string;
-  /**
-   * Primarily used for testing
-   * @default "https://uploadthing.com/api"
-   */
-  uploadthignApiUrl?: string;
-  /**
-   * Primarily used for testing
-   * @default "https://utfs.io/f"
-   */
-  uploadthingFileUrl?: string;
```

Comment:
these makes no sense as we don't allow this override to the `UTApi` anyways

Reply:
This option provides two benefits:

- Unit testing the driver internally without actually calling UTAPi but mocking it in unstorage end-to-end tests
- Allow users a way to Proxy your API if their server deployment requires this for any reason

I think both are internal and advanced cases. But if you really want to discourage users more to change them, we can definitely make it better with naming and JSDocs.

Reply:
Yes but that would only override the URL for the fetch calls that doesnt use utapi. All fetches made within utapi will still use uploadthing.com (although we have an internal env variable that overrides it)